src/styles.ts

```diff
@@ -121,7 +125,7 @@ export const styles = css`
 
 	:host([always-float-label]) input,
 	#input:not(:placeholder-shown) {
-		transform: translateY(var(--label-translate-y));
+		transform: translateY(var(--input-translate-y));
```

Comment:
Can you explain why this is necessary? It is not obvious to me.

Reply:
For this reason. I need the translation of the label+input block to be different when a user is actually typing in the input, otherwise I can't manage, with the current structure, to have the placeholder centered and then the input spaced as requested with the label inside the block.

https://github.com/Neovici/cosmoz-input/assets/122988480/56e8e6c9-a4fb-4964-a076-cd87131b1fb9

Reply:
For the textarea I needed to diversify the two elements, because, while in `cosmoz-input` there is only one row in all usages, a textarea is usually structured in more rows, and a percentage modification there would cause proportional spacing going more and more off while adding lines (cfr. screenshot), while the label would always be in the same position.

https://github.com/Neovici/cosmoz-input/assets/122988480/75c4e9a8-117d-4a83-bf5c-0018757607a0

Reply:
OK... but does it have to be a percentage value?

Reply:
Hm, for the `cosmoz-input` components it doesn't really play a role, since their size is always the same through cosmoz. I should check if a pre-determined value (e.g. 8px) can work the same way.